Azhao95 dd/grace permissions instructions (#28462)
DataDog/documentation: content/en/getting_started/integrations/_index.md

For information on how Datadog handles your data, and other security considerations, see the [Security documentation][30].

> ## Granular Access Control

Reviewer: ```suggestion
## Granular access control
```

> By default, access to integration resources (accounts, services, webhooks) is unrestricted. Granular access controls can be used to restrict the behavior of users, teams, roles, or your full organization at the integration resource level. Not all integrations support granular access controls yet. 

Author: I'd like to add a link out to a new page that doesn't exist yet, eventually-- should we wait to release this until there is something to link to?

Reviewer: Before I can answer your questions, I have a few questions in return:
1. How complex will this table be? i.e. how many columns?
2. How extensive will it get? i.e. how many rows?

In the meantime, you can do either (embed a table or [create a new page](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/4570644628/Add+content)).

Automation will likely involve devs. We can work with you on that if that ends up being the best route.

Author: @rtrieu Chatted with the integration teams, and proposing that each integration's documentation page gets updated with GRACE support. I think we can do this manually up front, for now, not worth investing in automation. 

I'd rather not introduce a table that lists supported integrations as another point of maintenance, and instead just point users to check the individual integration. Does that work?

Reviewer: Hi @azhao95-dd that makes sense, and that works. If things change, and we decide customers would benefit from a list of all the integrations that support GRACE, get in touch with us in #documentation, and we can discuss single-sourcing the page. Thanks!

> By default, access to integration resources (accounts, services, webhooks) is unrestricted. Granular access controls can be used to restrict the behavior of users, teams, roles, or your full organization at the integration resource level. Not all integrations support granular access controls yet. 

Reviewer: ```suggestion
By default, access to integration resources (accounts, services, webhooks) is unrestricted. Granular access controls can be used to restrict the behavior of users, teams, roles, or your full organization at the integration resource level.
```

We generally list what we do support, but do not list what we do not support. With that said, is GRACE for integrations in Preview? We could add an alert box that says that.

1. While viewing an integration, locate the resource that should have granular access controls applied. 
2. Click **Set Permissions**.

> 3. By default, everyone in your org has full access. Click **Restrict Access**. 

Author: How can I insert an image?

Reviewer: there's a lot of information on our [Screenshots and Video Confluence page](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1985314888/Screenshots+and+Videos), but essentially:
1. Upload image to correct directory - in this case it would be `~/static/images/getting-started/integrations`
2. Add image shortcode to this file and update the image filepath: {{< img src="path/to/your/image-name-here.png" alt="Your image description" style="width:100%;" >}}

> 4. The dialog box updates to show that members of your organization now have **Viewer** access by default.

Reviewer: ```suggestion
4. The dialog box updates to show that members of your organization have **Viewer** access by default.
```

> 5. Use the dropdown to select one or more teams, roles, or users that may edit the monitor. Note: The [Integrations Manage][45] permission is also required to edit individual resources.  

Reviewer: ```suggestion
5. Use the dropdown to select one or more teams, roles, or users that may edit the monitor.
    **Note**: The [Integrations Manage][45] permission is also required to edit individual resources.  
```

6. Click **Add**.
7. The dialog box updates to show the updated permissions

> 8. Click **Save**. The integration page will automatically refresh with updated permissions. 

Reviewer: ```suggestion
8. Click **Save**. The integration page automatically refreshes with updated permissions. 
```

**Note:** To maintain your edit access to the resource, the system requires you to include at least one role or team that you are a member of before saving.

To restore general access to a integration resource with restricted access, follow the steps below:
1. Click **Set Permissions**.
2. Click **Restore Full Access**.

> 3. Click **Save**. The integration page will automatically refresh with updated permissions. 

Reviewer: ```suggestion
3. Click **Save**. The integration page automatically refreshes with updated permissions. 
```
